### ISSUE TITLE
Neptune 4 Pro Support

### ISSUE THREAD

**Issue by iosdoctur** [Molodos/ElegooNeptuneThumbnails#16]
**Describe the feature you would like to have added: Neptune 4 Pro support
A clear and concise description of what you want to be added

**Describe why the above feature should be added:
Currently unable to view thumbnail on Neptune 4 Pro when using Cura
A clear and concise description of why the feature is relevant

**Additional information:**
Add additional information (optional)
Please if you can add it’ll be awesome!

**Comment by Molodos** [comment]
Good news: I am currently working on it. Expect the update to be available in less than a week :)

**Comment by iosdoctur** [comment]
Thank you for the quick response and that is great news, I’m definitely looking forward to it!

Thank you,
Haris

On Aug 25, 2023, at 3:00 AM, Michael Weichenrieder ***@***.***> wrote:

﻿

Good news: I am currently working on it. Expect the update to be available in less than a week :)

**Comment by Molodos** [comment]
Just published the update that adds neptune 4 support :)

**Comment by iosdoctur** [comment]
Michael!
Thank you so much man, I really appreciate the urgency on this, can’t tell you enough how grateful I am.I I recently bought a Neptune 4 pro and was not aware of how new it is in the market with little support. I just tested the plugin and it works great, love how you’ve given users the ability to customize what they want displayed in each corner. I also attached a preview of my printer working with the plugin.

Thank you,
Haris

On Aug 26, 2023, at 12:22 PM, Michael Weichenrieder ***@***.***> wrote:

﻿

Just published the update that adds neptune 4 support :)
